Add cross references to virtual machine docs (#8913)
istio/istio.io: content/en/docs/ops/diagnostic-tools/virtual-machines/index.md

This page describes how to troubleshoot issues with Istio deployed to Virtual Machines.
Before reading this, you should take the steps in [Virtual Machine Installation](/docs/setup/install/virtual-machine/).

> Additionally, understanding [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) can help understand how each component interacts.

Reviewer: ```suggestion
Additionally, [Virtual Machine Architecture](/docs/ops/deployment/vm-architecture/) can help you understand how the components interact.
```
